README.md

```diff
@@ -34,19 +33,23 @@ To setup the project locally read these wiki pages and follow the instructions:
  - [Fork, Clone and Remote](https://github.com/anitab-org/mentorship-backend/wiki/Fork%2C-Clone-%26-Remote)
  - [Export Environment Variables](docs/environment-variables.md)
 
-### Run app in Windows
+### Run app locally
 
-The project runs on Python 3.
+The project runs on Python 3.9. We use [Poetry](https://python-poetry.org/) to manage dependencies. It's powerful yet simple to use.
```

Comment:
@bartekpacia we should test it for other python versions as well. Python 3.9 is launched recently and not all users have migrated to it

Reply:
I agree, however, we shouldn't hold onto the old versions as well.
I will test on 3.8.6, 3.7.9 and 3.6.12

Reply:
Yes, just test them from python>=3.6

Reply:
@devkapilbansal done :)